boxfile.yml

```diff
@@ -105,20 +113,25 @@ worker.sidekiq:
     data.storage:
       - public/system
 
-  cron:
-    - id: generate_static_gifs
-      schedule: '*/15 * * * *'
-      command: 'bundle exec rake mastodon:maintenance:add_static_avatars'
 
+worker.cron_only:
+  start: sleep 365d
+
+  writable_dirs:
+    - tmp
+
+  log_watch:
+    rake: 'log/production.log'
+
+  network_dirs:
+    data.storage:
+      - public/system
+
+  cron:
     - id: update_counter_caches
       schedule: '50 * * * *'
       command: 'bundle exec rake mastodon:maintenance:update_counter_caches'
 
-    # runs feeds:clear, media:clear, users:clear, and push:refresh
-    - id: do_daily_tasks
-      schedule: '00 00 * * *'
-      command: 'bundle exec rake mastodon:daily'
-
     - id: clear_silenced_media
```

Comment:
Very subjective on whether you want to run this or not. Do you want to delete stuff shadowbanned people upload? Part of the appeal of the "silence" feature is that it's a more or less invisible quarantine for the person who's in it.

Reply:
I'll comment this one out and leave a note on when it might be desirable to enable it, then. :+1: